### DISCUSSION TITLE
Restructuring Website Slightly

### DISCUSSION THREAD

**Discussion by ArchieHickmott** [ArchieHickmott/quotebook#29] [Ideas]
How about we restructure the website a little bit so it will make more sense for when we split app.py up in to separate scripts and for the admin pages. Currently the site map looks like this:

<image src="https://github.com/user-attachments/assets/02c3cecc-c7b0-484a-8e0e-f62c78204069" width="400"> 

however as there are more "admin" features this admin page will be hard to manage so how about something more like this:

<image src="https://github.com/user-attachments/assets/e25fa02e-51f9-4a67-a75c-56aed56743f2" width="400"> 

that way as the slight grows it's a bit more organised

**Comment by Helios-fr** [comment]
I think we should rewrite the code to make it more readable and maintainable.

**Reply by ArchieHickmott** [reply]
I'll start of by getting rid of the ORM and using plain SQL
